### DISCUSSION TITLE
What is the accepted way to onboard a contact

### DISCUSSION THREAD

**Discussion by HoffmannP** [nmshd/feedback#26] [General]
I followed the instructions on https://enmeshed.eu/integrate/integration-example but still have on question: How should one proceed with the next relationship. Or put it another way: Should you create a new RelationshipTemplate (and hence QR code) for every user or how do you suggest one should distinguish multiple users on my platform with their enmashed profile?

**Comment by Milena-Czierlinski** [comment]
If you don't use personalized RelationshipTemplates, e.g. to create Attributes for the peer with CreateAttributeRequestItems, you can use the same RelationshipTemplate for multiple peers. In this case, make sure either not to set `maxNumberOfAllocations`  of the [RelationshipTemplate](https://enmeshed.eu/integrate/data-model-overview#relationshiptemplate) or to configure it appropriately. With the example Request from the [Integration example](https://enmeshed.eu/integrate/integration-example) using ShareAttributeRequestItems and ReadAttributeRequestItems this should work just fine. You can distinguish the users then by their responses to the ReadAttributeRequestItems.

**Comment by HoffmannP** [comment]
Currently our users authenticate via SSO from another platform that gives zero identifiable information. To receive first/last name and e-mail using enmeshed is advised. As I only have a generic uuid for the user I assume that the best approach is to create  new RT for every new user?

**Comment by tnotheis** [comment]
Even IDs count as PII under the GDPR. 

Also, if I understand you correctly, you want to share the user's name and email address. And that's **definitely** PII. So yes, personalized relationship templates are 100% the way to go in this scenario.
